NSS-Day-Cohort-55/tabloidfullstack-articulate-proficient-isopods
# Cc 25 view all users

Testing Instructions:

Fetch the code, run the program and after logging in, navigate to Users.
A list of users should generate.
Tell me I did a good job.
Exit app.
